distribution/pull.go

```diff
@@ -147,7 +147,7 @@ func Pull(ctx context.Context, ref reference.Named, imagePullConfig *ImagePullCo
 			return TranslatePullError(err, ref)
 		}
 
-		imagePullConfig.ImageEventLogger(ref.String(), repoInfo.Name(), "pull")
+		imagePullConfig.ImageEventLogger(reference.FamiliarString(ref), reference.FamiliarName(repoInfo.Name), "pull")
```

Comment:
Yes it should, though not sure if I intentionally left it it out since it mentioned endpoints. Either way this change should not change such behavior.